action-scripts/brownie/scripts/pokeInjectors.py

```diff
@@ -3,35 +3,64 @@
 import os
 
 
+CHAINS_TO_KEEP = [
+    "zkevm-main",
+    "gnosis-main",
+    "mode-main",
+    "linea-main",
+    "fraxtal-main",
+]
+
+
 def main():
     ## LOAD wallet
+    errors = False
     mnemonic = os.environ["KEYWORDS"]
     account = accounts.from_mnemonic(mnemonic)
     print(f"Keeper Address: {account.address}")
-    ## ZKEVM
-    network.disconnect()
-    network.connect("zkevm-main")
-    book = AddrBook("zkevm")
-    injectors = book.extras.maxiKeepers.gaugeRewardsInjectors.values()
-    for injectorAddress in injectors:
-        injector = Contract(injectorAddress)
-        (ready, performdata) = injector.checkUpkeep(b"")
-        if ready:
-            injector.performUpkeep(performdata, {"from": account})
-        else:
-            print(f"ZKEVM: {injector.address} not ready")
-    ## Gnosis
-    network.disconnect()
-    network.connect("gnosis-main")
-    book = AddrBook("gnosis")
-    injectors = book.extras.maxiKeepers.gaugeRewardsInjectors.values()
-    for injectorAddress in injectors:
-        injector = Contract(injectorAddress)
-        (ready, performdata) = injector.checkUpkeep(b"")
-        if ready:
-            injector.performUpkeep(performdata, {"from": account})
-        else:
-            print(f"GNOSIS: {injector.address} not ready")
+    for chain in CHAINS_TO_KEEP:
+        try:
+            network.disconnect()
+        except:
+            pass
+        try:
+            network.connect(chain)
+        except:
+            print(f"Skipping {chain} because no network can be found")
+            continue
+        try:
+            book = AddrBook(chain.replace("-main", ""))
+        except:
+            print(
+                f"Skipping {chain} because no Address Information can be found in bal_addresses"
+            )
+            continue
+        to_poke = []
+        try:
+            to_poke = book.extras.maxiKeepers.gaugeRewardsInjectors.values()
```

Comment:
```suggestion
            to_poke += book.extras.maxiKeepers.gaugeRewardsInjectors.values()
```
@gosuto-inzasheru if anything this is the bug  the way I see it.